validator/lib/validateTileset.js

```diff
@@ -173,3 +174,14 @@ function validateTileHierarchy(root, tilesetDirectory) {
             return message;
         });
 }
+
+function errorMessage(originalMessage, tile) {
+    if (defined(tile)) {
```

Comment:
Related to the note above, it's fine for this helper function to always assume a `tile` is passed in.